doc/gov/chapters/chapter02.md

```diff
@@ -5,43 +5,53 @@
 
 
 
 <a name="2.2"></a>
-## 2.2 Request Membership
+## 2.2 Request Onboarding Info Packet
 
 [Welcome Message Here]
 
 - **Guide To Getting Started**
 
   - **How do I start the process of joining?** Onboarding starts as soon as you submit a request to join CNTT via email to xxxx@xxxxx.com.
   
-  - **How long will it take to onboard?** The onboarding process will take approximately X hours of your time initially.
+  - **How long will it take to onboard?** The onboarding process will take approximately one hour of your time initially.
   
   - **What will I learn about during onboarding process?** You’ll get an overview of the CNTT mission and goals, community organizational structures and processes, as well as technology development status and roadmaps. Additionally, you’ll be made aware of participant responsibilities and legal obligations. Finally, you’ll be provided with support and logistical information you’ll need to join the community.
 
-  - **What help will I get throughout the onboarding process?** Once you submit a request to join the Task Force, during the onboarding process members are supported by the CNTT onboarding team members: (resource contact info/emails here) indicated in your “Getting Started” email. Onboarding members are welcomed to reach out for assistance any time during the onboarding process.
+  - **What help will I get throughout the onboarding process?** Once you submit a request to participate in the Task Force, during the onboarding process participants are supported by the CNTT onboarding team members: indicated in your “Getting Started” email. Onboarding participants are welcomed to reach out for assistance any time during the onboarding process.
   
   - **What happens once I get onboarded?** Once you have reviewed the CNTT onboarding information and have agreed to the perquisite terms and agreements associated with participating and contributing to the Task Force, and you have completed onboarding and now it will be your opportunity to join desired work streams. For an initial period after onboarding, you will be assigned a volunteer mentor from the Task Force to help you get up to speed and contributing quickly.
   
   - **Diagram**
-<p align="middle"><img src="../figures/onboarding_proc.png" alt="onboarding_proc" title="Onboarding Process" width="60%"/></p>
+<p align="middle"><img src="https://github.com/cntt-n/CNTT/tree/master/doc/gov" alt="onboarding_proc" title="Onboarding Process" width="60%"/></p>
  
 <a name="2.3"></a>
 ## 2.3 Prerequisites
 
+CNTT has no formal membership, and nothing to sign. Participation is open to anyone, whether you are an employee of an LFN member company, supporter, or observer. By participating, you do automatically accept the individual anti-trust policies of LFN and GSMA, the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy. 
+
+LFN Anti-Trust Policy https://r.lfnetworking.org/Antitrust%20Slide.pdf
```

Comment:
in the current document view it is consolidated into a single paragraph.  from this view, it looks like the intent is that it should be formatted as a markdown list.  I think this might also take care of the ToR first use that @pgoyal01 identified; as it's explained in the list, but perhaps not very readable in the current form.

Reply:
@kedmison I created issue #823 to address the individual vs. company item. Also, not sure what you mean by 'format as a markdown list'. I'm going to push it and let me know if separate issue for the markdown list needs to be created. Thanks for the help reviewing.

Reply:
Here is a snapshot of how the list of policies shows in presentation view of the document.  I think you may want to structure these as a list so that the policies and URLs match the structure in the raw document.

![image](https://user-images.githubusercontent.com/810497/71284572-073c8d80-2331-11ea-9c08-53621b90b3c6.png)